examples/client/Locomotion/src/pages/ActiveRide/index.js

```diff
@@ -108,13 +108,13 @@ const RidePage = ({ mapSettings, navigation }) => {
   };
 
   const backToMap = () => {
-    if (currentBsPage === BS_PAGES.SERVICE_ESTIMATIONS) {
+    if (currentBsPage === BS_PAGES.SERVICE_ESTIMATIONS
```

Comment:
1. extract to array of screens.
2. in case of the active ride, maybe we can send a param to the function